tests/rustdoc-ui/issue-108653.rs

```diff
@@ -0,0 +1,35 @@
+// This is ensuring that the UI output for associated items is as expected.
```

Comment:
Please put both these tests under `intra-doc` with a more descriptive file name.

Reply:
I tend to prefer using the issue number which will provide more information rather than a longer file name which will still be lacking.

Moving the files in the sub directory. :+1:

Reply:
you can include both, `issue-108653-associated-items.rs`

Reply:
Good idea, thanks!